src/avatar/avatar.js

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import React from 'react';
+
+import {getOverrides} from '../helpers/overrides';
+
+import {Avatar as StyledAvatar, Root as StyledRoot} from './styled-components';
+import type {PropsT, StateT} from './types';
+
+export default class Avatar extends React.Component<PropsT, StateT> {
+  static defaultProps: $Shape<PropsT> = {
+    overrides: {},
+    size: 'scale1000',
```

Comment:
we should be able to import current Theme applied. In this case we would be able to reference exact value instead of string name. Need to think about better way to do it
cc @nadiia @gergelyke @austin94

Reply:
Do you mean customers import the `theme`? We shouldn't encourage doing it in order to set the size here. 
The current approach is good as an initial implementation. What we can do further to allow custom sizes is to set the default `size` prop to `scale1000` and allow any string as a value of `size` prop, for example '25px'. Then in the `styled-components.js`:
```
 function getSize(props: StylePropsT) {
  const {$size, $theme} = props;
   return $theme.sizing[$size] || $size;
}
```

Reply:
if we were able to put `size:  currentTheme.scale.scale1000` here, that would help to not use this string constant. Strings to refer to object property are vague and can be easily compromised. It seems to be not the first example where we try to address it

Reply:
if importing `theme` is not allowed for some reasons, we can think about how to accommodate it still. For instance, freeze theme object so it would be available only for reading. 
@nadiia  Can you elaborate, why we shoulnd't import theme?

Reply:
this api follows conventions set in [Block](https://github.com/uber-web/baseui/blob/master/src/block/examples.js#L71). i can update the interface to allow arbitrary sizes like `10px`, `1rem`, etc. does that solves both of our issues? we should consider exposing a withTheme context consumer or HOC like [here](https://github.com/uber-web/baseui/blob/e3d2c136b2dcbdee8f113adbd3a8af755f2b10a0/src/styles/styled.js#L14).